### ISSUE TITLE
Dropdown and navigation tab order problems

### ISSUE THREAD

**Issue by dimitur2204** [gpbl/react-day-picker#2785]
There is an issue when using `captionLayout="dropdown"`. It has to do with tab order, where if you still have navigation that gets focused first since its first in the HTML tree. It is a bit confusing since taborder looks and feels a bit off.

## To reproduce

https://codesandbox.io/p/sandbox/ftggph

```tsx
import { DayPicker } from "react-day-picker";
import "react-day-picker/style.css";

export default function App() {
  return <DayPicker
  captionLayout="dropdown"
  defaultMonth={new Date(2024, 6)}
  startMonth={new Date(2024, 6)}
  endMonth={new Date(2025, 9)}
/>;
}
```

CodeSandbox: https://codesandbox.io/p/sandbox/ftggph

## Actual Behavior

When you start tabbing through you first get focus on the right navigation arrows and after that on the dropdown on the left.

## Expected Behavior

Tab order should be left-to-right top-to-bottom. And in this case first on the two dropdowns and later on the 

## Screenshots

![Image](https://github.com/user-attachments/assets/85a73018-cd88-4ab8-a9bf-e9cb3831b9e2)
